src/lakefs_spec/client_helpers.py

```diff
@@ -63,3 +63,8 @@ def revert(client: LakeFSClient, repository: str, branch: str, parent_number: in
     client.branches_api.revert_branch(
         repository=repository, branch=branch, revert_creation=revert_creation
     )
+
+
+def rev_parse(client: LakeFSClient, repository: str, ref: str, parent: int = 0) -> str:
```

Comment:
Should we validate client-side and raise a `ValueError` here if `parent` is negative?

Reply:
Good idea